README.md

```diff
@@ -87,27 +66,43 @@ risks introduced are outweighed by the functionalities they support.
 
 ![image](/images/install05.png)
 
-### Authentication with PAT (optional)
-Create a Personal Access Token (PAT) for authentication and save the token value as a repository secret. Review [Personal Access Token (PAT) Requirements and Risks](#personal-access-token-pat-requirements-and-risks) before using a PAT.
+### Authentication with Fine-grained PAT (optional)
+Scorecard can run successfully with the workflow's default `GITHUB_TOKEN`.
+However, the `Branch-Protection` and (experimental) `Webhooks` checks require additional data that isn't accessible with that token.
 
-1. [Create a Personal Access Token](https://github.com/settings/tokens/new?scopes=public_repo,read:org,read:repo_hook,read:discussion) with the following read permissions:
-    - Note: `Token for OSSF Scorecard Action - myorg/myrepo` (Note: replace `myorg/myrepo` with the names of your organization and repository so you can keep track of your tokens.)
-    - Expiration: `No expiration`
-    - Scopes:
-        * `repo > public_repo`                  Required to read [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) settings. **Note**: for private repositories, you need scope `repo`.
-        * `admin:org > read:org`                Optional: not used in current implementation.
-        * `admin:repo_hook > read:repo_hook`    Optional: needed for the experimental [Webhook](https://github.com/ossf/scorecard/blob/main/docs/checks.md#webhooks) check.
-        * `write:discussion > read:discussion`  Optional: not used in current implementation.
+We therefore suggest you create a fine-grained Personal Access Token (PAT) that Scorecard may use for authentication.
 
-![image](/images/tokenscopes.png)
+1. [Create a fine-grained Personal Access Token](https://github.com/settings/personal-access-tokens/new) with the following settings:
+    - Token name: `OpenSSF Scorecard Action - $USER_NAME/$REPO_NAME>`
+      (Note: replace `$USER_NAME/$REPO_NAME` with the names of your organization and repository so you can keep track of your tokens.)
+    - Expiration: Set `Custom` and then set the date to exactly a year in the future (the maximum allowed)
+    - Repository Access: `Only select repositories` and select the desired repository. 
+      Alternatively, set `All repositories` if you wish to use the same token for all your repositories.
+    - Repository Permissions:
+        * `Administration: Read-only`: Required to read [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) settings.
+        * `Metadata: Read-only` will be automatically set when you set `Administration`
+        * `Webhooks: Read-only`: (Optional) required for the experimental [Webhook](https://github.com/ossf/scorecard/blob/main/docs/checks.md#webhooks) check.
```

Comment:
@laurentsimon What is keeping the webhook check experimental? Is it users not wanting to give a token admin permissions? Does fine grained help with this?

Reply:
chatted offline. Nothing is keeping it experimental. We started off as experimental to allow for breaking changes. But the check has been stable for a while. SO we should be able to make it non-experimental